terraform/20-app/eventbridge.scheduled-scaling.tf

```diff
@@ -0,0 +1,66 @@
+locals {
+  db_scale_up_cron_expression   = "cron(30 07 ? * MON-FRI *)"
+  db_scale_down_cron_expression = "cron(10 20 ? * MON-FRI *)"
+}
+
+module "eventbridge_scheduled_scaling" {
+  source  = "terraform-aws-modules/eventbridge/aws"
+  version = "3.17.1"
+  create  = local.is_scaled_down_overnight
+
+  create_bus         = false
+  create_role        = true
+  role_name          = "${local.prefix}-eventbridge-scheduled-scaling-role"
+  attach_policy_json = true
+  policy_json = jsonencode({
+    Version = "2012-10-17",
+    Statement = [
+      {
+        Effect = "Allow",
+        Action = [
+          "rds:StopDBCluster",
+          "rds:StartDBCluster",
+        ],
+        Resource = [
+          module.aurora_db_app.cluster_arn,
+          module.aurora_db_feature_flags.cluster_arn,
+        ]
+      }
+    ]
+  })
+
+  schedules = {
+    # Scale up actions
+    "${local.prefix}-scheduled-start-app-db" = {
+      schedule_expression      = local.db_scale_up_cron_expression
+      timezone                 = local.timezone_london
+      use_flexible_time_window = false
+      arn                      = "arn:aws:scheduler:::aws-sdk:rds:startDBCluster"
+      input = jsonencode({ DbClusterIdentifier = module.aurora_db_app.cluster_id })
+    }
+    "${local.prefix}-scheduled-start-feature-flags-db" = {
+      schedule_expression      = local.db_scale_up_cron_expression
+      timezone                 = local.timezone_london
```

Comment:
With eventrbidge scheduler we can specify a timezone, which we could not do with rules/targets.
This helps us because standard time is UTC so we'd have had to either manually calculate the time or come back and change it twice a year due to daylight savings